pom.xml

```diff
@@ -334,27 +334,27 @@
 			<dependency>
 				<groupId>ca.uhn.hapi</groupId>
 				<artifactId>hapi-base</artifactId>
-				<version>2.0</version>
+				<version>2.3</version>
```

Comment:
I think the closes you can upgrade to is 2.1 because of the [error source](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/hl7/impl/HL7ServiceImpl.java#L66) .
You can refer to [this doc](http://javadox.com/ca.uhn.hapi/hapi-base/2.1/ca/uhn/hl7v2/app/MessageTypeRouter.html)

@dkayiwa, would this be good enough ?

Reply:
Ok!! let me at-least try it locally and see if the build passes

Reply:
@dkayiwa @Ruhanga I have a build success after changing the upgrade to 2.1 so I guess am pushing these changes